libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -106,6 +106,21 @@
         }
       }
     }
+    &--primary-rounded {
```

Comment:
why create a variant when you just had to call `primary` & `rounded` props, would be top to be based on #8362